check rpm
not building docs (need oxygen)

8/19/2010 - created by Qing He <qing.he@intel.com>

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 24234db..69413e4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -50,6 +50,10 @@ MESSAGE(STATUS "Enabling multi dist support")
 ADD_DEFINITIONS( -DMULTI_SEMANTICS)
 ENDIF ( MULTI_SEMANTICS )
 
+FIND_PACKAGE(PkgConfig REQUIRED)
+PKG_CHECK_MODULES(RPM REQUIRED rpm)
+INCLUDE_DIRECTORIES( ${RPM_INCLUDE_DIRS} )
+
 IF ( NOT DEBIAN )
 FIND_LIBRARY(RPMDB_LIBRARY NAMES rpmdb)
 IF ( NOT RPMDB_LIBRARY )
@@ -159,22 +163,7 @@ ADD_SUBDIRECTORY(tools)
 ADD_SUBDIRECTORY(applayer)
 ADD_SUBDIRECTORY(tests)
 ADD_SUBDIRECTORY(examples)
-ADD_SUBDIRECTORY(doc)
-
-FIND_PACKAGE(SWIG)
-
-IF ( SWIG_FOUND )
-  ADD_SUBDIRECTORY(bindings)
-ELSE ( SWIG_FOUND )
- # for older version of swig/cmake
- FIND_PROGRAM(SWIG_EXECUTABLE
-   NAMES swig-1.3 swig
-  PATHS ${SWIG_DIR} ${SWIG_DIR}/.. ${SWIG_DIR}/../../bin /usr/bin /usr/local/bin ${CMAKE_INSTALL_PREFIX}/bin
- )
- IF ( SWIG_EXECUTABLE )
-  ADD_SUBDIRECTORY(bindings)
- ENDIF ( SWIG_EXECUTABLE )
-ENDIF ( SWIG_FOUND )
+#ADD_SUBDIRECTORY(doc)
 
 MESSAGE(STATUS "version: ${VERSION}")
 
